addons/sourcemod/scripting/freak_fortress_2.sp

```diff
@@ -8251,6 +8237,17 @@ public Action:Timer_UseBossCharge(Handle:timer, Handle:data)
 	return Plugin_Continue;
 }
 
+stock RemoveShield(client, attacker, Float:position[3])
+{
+	TF2_RemoveWearable(client, shield[client]);
+	EmitSoundToClient(client, "player/spy_shield_break.wav", _, _, _, _, 0.7, _, _, position, _, false);
+	EmitSoundToClient(client, "player/spy_shield_break.wav", _, _, _, _, 0.7, _, _, position, _, false);
+	EmitSoundToClient(attacker, "player/spy_shield_break.wav", _, _, _, _, 0.7, _, _, position, _, false);
+	EmitSoundToClient(attacker, "player/spy_shield_break.wav", _, _, _, _, 0.7, _, _, position, _, false);
+	TF2_AddCondition(client, TFCond_Bonked, 0.1); // is this even needed?
```

Comment:
Get rid of the comment and I'll merge.
(or alternatively update it to indicate that this line is there to show the "MISS!" line)